ui/src/main/webapp/app/query/input-manager/input-manager.html

```diff
@@ -1,21 +1,25 @@
 <div layout-margin ng-show="ctrl.primaryInput" ng-cloak>
+    <md-checkbox ng-if="!ctrl.first" ng-model="ctrl.usePreviousOutput" ng-change="ctrl.onCheckboxChange()" aria-label="use output from previous operation">
+        Use output from previous operation
+    </md-checkbox>
     <div ng-switch="ctrl.primaryInput">
```

Comment:
When the "Use output from previous operation" is selected, perhaps we could either hide the input box and show a message "using output from previous operation" or just change the placeholder for the input component.